apps/webapp/app/components/primitives/Buttons.tsx

```diff
@@ -258,13 +267,15 @@ export function ButtonContent(props: ButtonContentPropsType) {
               )}
             />
           ))}
-        {shortcut && (
-          <ShortcutKey
-            className={cn(shortcutClassName)}
-            shortcut={shortcut}
-            variant={variation.shortcutVariant ?? "medium"}
-          />
-        )}
+
+        {shortcut &&
+          (!props.shortcutPosition || props.shortcutPosition === "after-trailing-icon") && (
```

Comment:
It would be better to assign a default to `shortcutPosition` than do this. That way it will always have a value avoiding an undefined check here